packages/cyberstorm/src/components/Avatar/Avatar.module.css

```diff
@@ -1,19 +1,31 @@
 .root {
+  display: flex;
   align-items: center;
   justify-content: center;
-  border-radius: var(--size) !important;
-  background-color: #fff;
+  border-radius: calc(var(--size) * 4.92) !important;
+  background: transparent;
 }
 
 .image {
-  min-width: var(--size);
-  max-width: var(--size);
-  min-height: var(--size);
-  max-height: var(--size);
-  border-radius: var(--size);
+  width: var(--size);
+  height: var(--size);
+  border-radius: calc(var(--size) * 4.92);
   transition: opacity 150ms ease-in-out;
 }
 
+.image:where(.placeholder) {
```

Comment:
IDK what's the status with global CSS buit should we use it for `#39e9aa` and `line-height` here?